integrations/stripe.ts

```diff
@@ -1,15 +1,96 @@
 import Stripe from "stripe"
 import { str, envsafe } from "envsafe"
+import { User } from "db"
 
 export const env = envsafe({
   DOMAIN: str(),
   STRIPE_WEBHOOK_SECRET: str(),
   STRIPE_SECRET_KEY: str(),
 })
 
-export * from "stripe"
+// https://stackoverflow.com/questions/59903956/how-to-use-stripe-types-in-typescript
 
-// Centrally configure an instance of the Stripe client
-export default new Stripe(env.STRIPE_SECRET_KEY, {
+const getStripeSecretKey = (): string => {
+  const secretKey = env.STRIPE_SECRET_KEY
+  if (!secretKey) {
+    throw new Error("Stripe secret key not found in variable 'env.STRIPE_SECRET_KEY'")
+  }
+  return secretKey
+}
+
+const stripe = new Stripe(getStripeSecretKey(), {
   apiVersion: "2020-08-27",
+  typescript: true,
```

Comment:
Is typescript a valid part of this config object? I don't see it here https://github.com/stripe/stripe-node#initialize-with-config-object